src/index.js

```diff
@@ -1,14 +1,46 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
[OPTIONAL] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.